### PR TITLE
Closes #1 - Create initial index.html markup

### LINKED ISSUE #1: Create initial index.html markup

### PR DESCRIPTION
**Pull request by sotnyk-lv**

### CODE REVIEW

**Review by bmtfc — CHANGES_REQUESTED** [review]
Change UTF-8 to UTF-16

File: initial.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (GilgameshUA) [inline initial.html:6]: Add Title

File: initial.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+</head>
+<body>
+
```
Comment (marko-yaciv) [inline initial.html:9]: Everything is good but I need to reject))

**Review by marko-yaciv — CHANGES_REQUESTED** [review]

**Review by Cv287 — CHANGES_REQUESTED** [review]
We need changes!!!

File: initial.html (hunk 2 above)
Comment (SnowfallR2) [inline initial.html:9]: Remove extra space between bodies

**Review by IhorVodko — CHANGES_REQUESTED** [review]
Change scale

**Review by marko-yaciv — APPROVED** [review]

**Review by IhorVodko — APPROVED** [review]

**Review by bmtfc — APPROVED** [review]
gj

**Review by nullStandard — APPROVED** [review]

**Review by Cv287 — APPROVED** [review]

**Review by GilgameshUA — APPROVED** [review]

**Review by SnowfallR2 — APPROVED** [review]

**Pull request merged**